index.html

```diff
@@ -49,8 +42,199 @@ <h1>I'm Alex. <br>
                 <li><a href="#" class="github" ><img src="images/github.svg" alt=""></a></li>
                 <li><a href="#" class="vectorsixer" ><img src="images/Vectorsixer.png" alt=""></a></li>
                </ul>
+
+               <!--Work section-->
+               <div id ="bg-work1">
```

Comment:
- [ ] Well said 😄  in this comment, `work section`, then  please make sure you wrap your projects in a section container as said the requirement, it is more semantically appropriate in HTML best practice
- [ ] after your card are warped in one section container, kindly use `display grid` to the section class to display its contents (projects)

Reply:
the div is for the header image,the section is displayed below it